tutorials/02_installation.md

```diff
@@ -36,26 +36,41 @@ sudo apt-get install libignition-physics3-dev
 
```

Comment:
We should remove `libignition-physics3-dev` from here because that doesn't exist yet. Instead, how about the generic `libignition-physics<N>-dev` with instructions for `<N>` like we have on other libs?